src/monodroid/jni/dylib-mono.h

```diff
@@ -312,6 +314,7 @@ struct DylibMono {
 
 	monodroid_mono_domain_foreach_fptr                      mono_domain_foreach;
 	monodroid_mono_thread_create_fptr                       mono_thread_create;
+	monodroid_mono_thread_current                           mono_thread_current;
```

Comment:
This line should be *last* in the struct, as per the comment at the beginning of the struct.

Reply:
Doh, I missed that comment. It is needed for ABI compatibility I guess. Fixed.